packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -668,14 +680,6 @@ export class GumbyController {
 
             case ConversationState.WAITING_FOR_TRANSFORMATION_OBJECTIVE: {
                 const objective = data.message.trim().toLowerCase()
-                // since we're prompting the user, their project(s) must be eligible for both types of transformations, so track how often this happens here
-                if (objective === 'language upgrade' || objective === 'sql conversion') {
-                    telemetry.codeTransform_submitSelection.emit({
```

Comment:
Better to emit this above since those functions are used in multiple places, so remove it from here.